opencti-platform/opencti-graphql/src/types/stix-2-0-common.d.ts

```diff
@@ -19,8 +19,10 @@ export interface StixObject {
   x_opencti_granted_refs?: string[]
   x_opencti_type: string;
   type: string;
+  // custom
   x_opencti_workflow_id?: string;
   x_opencti_files: Array<StixFile>;
+  x_opencti_modified_at: StixDate;
```

Comment:
is it mandatory ? I wonder if a ? is not missing here like
```suggestion
  x_opencti_modified_at?: StixDate;
```